tests/test_panels/test_session.py

```diff
@@ -0,0 +1,390 @@
+from pyramid import testing
+from pyramid.request import Request
+
+try:
+    from pyramid.session import SignedCookieSessionFactory as Sessn
+except ImportError:
+    # pyramid_debugtoolbar supports Pyramid==1.4
+    from pyramid.session import UnencryptedCookieSessionFactoryConfig as Sessn
```

Comment:
I'd be fine bumping the minimum supported version if it helps.

Reply:
However, doing that might actually make more work for you as we have to go through and remove dead code. So maybe this is better for now.